apps/pie-docs/src/components/assistive-text/assistive-text.md

```diff
@@ -0,0 +1,69 @@
+---
+eleventyNavigation:
+    key: 'Assistive Text'
+    parent: Components
+---
+
+## Overview
+
+Assistive text helps users understand what kind of information they should enter, how to format it, or any specific requirements related to the input. It can provide examples, suggestions, error messages, validation rules, or any other relevant details that aid in accurate and meaningful input.
+
+The purpose of assistive text is to enhance the user experience by reducing confusion, improving input accuracy, and ensuring users provide the expected information in the desired format.
+
+
+{% contentPageImage {
+    src:"../../assets/img/components/assistive-text/overview.svg",
+    alt: "Assistive text that is highlighting an error underneath a text input."
+} %}
+
+---
+
+## Resources
+
+{% notification {
+  type: "warning",
+  message: "We’re currently working on updating our Assistive Text documentation, please see the resources below."
+} %}
+
+{% resourceTable {
+    rows: [
+        {
+            resource: resourceTypes.COMPONENT,
+            link: "https://www.figma.com/file/R2rBfzJP0hG0MZorq6FLZ1/%5BCore%5D-Components-%E2%9A%AA%EF%B8%8F-%5BPIE-2.0%5D?type=design&node-id=981-5354&mode=design",
```

Comment:
Not necessarily one to include in this PR, but could we make all the external docsite links here open in a new window? Just a bit more usable that way. (Maybe in a followup PR, or separate ticket?)

Reply:
ah yeah I agree, would be good to do - will do this in a followup PR 😄 thaanks!